src/transformers/work.js

```diff
@@ -100,7 +100,7 @@ function handleCoverUrlRequests(dataUrls, result, context) {
   }
 
   const promises = requests.map(req =>
-    context.request('http:' + req.url, {encoding: null}).then(o => {
+    context.request(`${req.url.search('http') === -1 && 'http:' || ''}${req.url}`, {encoding: null}).then(o => {
```

Comment:
Her ville jeg nok skrive `req.url.startsWith('http') ? '' : 'http:'`, - men det er en smagssag.
Ville også lige nævne, at når vi opdaterer de automatisk genererede unit-tests, kommer der automatisk en test-case for dette :) (regenerering af unit-testene fandt denne fejl)